### DISCUSSION TITLE
My way of configuring neovim broke with the new refactor

### DISCUSSION THREAD

**Discussion by longregen** [NotAShelf/nvf#344] [Q&A] [answered]
I used to do this: https://notashelf.github.io/nvf/index.xhtml#ch-standalone-nixos

But, `neovimConfiguration` seems to be gone now (building error says "error: attribute 'neovimConfiguration' missing"). Am I missing something? Which is the new preferred way to  reconfigure nvf?

Thanks and sorry if the question is too basic, I'm relatively new to nix.

**Comment by NotAShelf** [comment]
Looks like there has been a mistake while @diniamo was moving `neovimConfiguration`. 

For now, please try using `inputs.<name of your nvf input>.lib.nvim.neovimConfiguration`

**Comment by diniamo** [comment, marked as answer]
Sorry, #345 should fix it

**Reply by longregen** [reply]
Wow, thank you so much for the quick fix! And thank you for providing `nvf` :heart:
